README.md

```diff
@@ -0,0 +1,61 @@
+# Space
```

Comment:
I think this file is great for now 💯  We should consider moving experiments/PoCs to a separate section once we get more requirements.